docs/index.md

```diff
@@ -3,15 +3,16 @@ title: InvenTree
 template: home.html
 hide:
     - toc
-    - navigation
 ---
 
 ## Intuitive Inventory Management
 
---------------
 
-{% with directory="indexgallery", per_page=1 %}
-{% include "carousel.html" %}
-{% endwith %}
+InvenTree is an open-source inventory management system which provides intuitive parts management and stock control. 
 
---------------
+
+It is designed to be lightweight and easy to use for SME or hobbyist applications, where many existing stock management solutions are bloated and cumbersome to use. However, powerful business logic works in the background to ensure that stock tracking history is maintained, and users have ready access to stock level information. InvenTree is designed to allow for a flexible installation. 
```

Comment:
@SchrodingersGat maybe we should remove the line about other solutions beeing bloated - talking bad about others seems to be bad style to me.

Reply:
Good point, I have updated the index page now..

Reply:
@SchrodingersGat while I understand (and mostly agree with) your point I think it might come off wrong. Especially if user from partkeepr etc. see it.